### PR TITLE
Cu 86bwmczwm/total group savings endpoint

### PR DESCRIPTION
**Pull request by Don-christo**
**Description of PR**
The creation of a group where a group of individuals come together and contribute some amount of money for a particular purpose.
_A brief description of what changes your PR introduces. The better this description, the quicker we can review, exchange
feedback, and merge!_

---

Please check all that apply.

#### Checklist:

- [x] Rebased with master
- [x] PR Description provided
- [x] Did you communicate these changes with the affected teams?
- [ ] Did unit tests/integration tests/E2E tests get properly added/updated?
- [ ] Is this change production-ready including testing, feature toggles, configurations, and deployment considerations?

#### Frontend Checklist:

- [ ] Is this responsive and mobile friendly?
- [ ] Did you follow the [ARIA specs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)

#### Backend Checklist:

- [x] Does the endpoint return the correct success/error format?

---

**Screenshots**  
_add relevant screenshots here_

---

**Test Coverage**  
_add relevant screenshots here_

### CODE REVIEW

**Comment by KristofaJosh** [comment]
Task linked: [CU-86bwmczwm Total group savings endpoint](https://app.clickup.com/t/86bwmczwm)

**Review by pelzman — COMMENTED** [review]
Be consistent with the usage of the HTTP-Status-Code you imported...

Good job so far

**Review by pelzman — COMMENTED** [review]
You branch includes another commit(rechart type)  which is not yours and it shouldn't be so, since you didnt work on it .

File: apps/client/package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "@types/react": "^18.2.37",
     "@types/react-dom": "^18.2.15",
     "@types/react-helmet": "^6.1.9",
+    "@types/recharts": "^1.8.29",
     "@typescript-eslint/eslint-plugin": "^6.12.0",
     "@typescript-eslint/parser": "^6.12.0",
     "@vitejs/plugin-react-swc": "^3.5.0",
```
Comment (KristofaJosh) [inline apps/client/package.json:40]: Your PR has changed here, why?

File: apps/server/src/controllers/groupControllers/createGroup.ts
Comment (KristofaJosh) [inline apps/server/src/controllers/groupControllers/createGroup.ts:1]: code can be improved

**Review by KristofaJosh — CHANGES_REQUESTED** [review]
I don't think your code is rebased

**Comment by KristofaJosh** [comment]
Delete your local branch
check into this branch
make the fixes

- document your API
- test your code (currently not working)
- make any fixes and improvements
- update your PR description

**Review by KristofaJosh — APPROVED** [review]

**Comment by KristofaJosh** [comment]
image

Update your documentation.
remove those highlighted values.

**Review by Jidekas — APPROVED** [review]
Everything works as intended. Though you could have stated something about the folder structure change in your PR description. makes it easy for the reviewer.
Great job!!

**Pull request merged**